packages/react-components/react-spinner/src/components/Spinner/Spinner.test.tsx

```diff
@@ -21,17 +21,12 @@ describe('Spinner', () => {
 
   it('has role progressbar', () => {
     const result = render(<Spinner label="Default Spinner" />);
-    expect(result.queryByRole('progessbar')).toBeDefined();
+    expect(result.queryByRole('progressbar')).toBeDefined();
   });
 
   it('renders Spinner with a label', () => {
     const result = render(<Spinner label="Loading" />);
     expect(result.getByText('Loading')).toBeDefined();
-    expect(result.queryByRole('progessbar')).toBeDefined();
-  });
-
-  it('doesnt render svg when status is inactive', () => {
```

Comment:
Since we provide two examples for uses to achieve this result in the docs can you add tests for those to verify they work as expected?